tests/phpunit/test/Translations/TranslationsTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Tests\test\Translations;
+
+use PHPUnit\Framework\TestCase;
+
+class TranslationsTest extends TestCase
+{
+
+    private const FILES = [
+      'messages.bg.php',
+      'messages.en.php',
+      'validators.bg.php'
+    ];
+
+    public function load(string $file): array
+    {
+        $translations = require 'translations/' . $file;
+        /**
+         * $duplicates variable is loaded from the translation file
+         * @var array $duplicates
+         */
+        $allowed = isset($duplicates) ? $duplicates : [];
+
+        return [
+            'translations' => $translations,
+            'allowed' => $allowed,
+        ];
+    }
+
+    /**
+     * For readability, translation messages should be sorted alphabetically.
+     * Sort the existing list by keys and compare it with the original.
+     */
+    public function testArrangement(): void
+    {
+        foreach (self::FILES as $file) {
+            $trans = $this->load($file);
+
+            $sorted = $trans['translations'];
+            ksort($sorted);
+            $this->assertSame($sorted, $trans['translations'], "The file '" . $file . "' is not sorted alphabetically");
+        }
+    }
+
+    /**
+     * Test for duplicate keys
+     */
+    public function testDuplicateKeys(): void
```

Comment:
The order test should be enough.
We can remove this `testDuplicateKeys` keys and merge the changes

Reply:
Sure